server/src/test/java/com/google/coffeehouse/servlets/CreateClubServletTest.java

```diff
@@ -43,38 +44,42 @@
  */
 public class CreateClubServletTest {
   private static final String NAME = "Club Name";
+  private static final String OWNER_ID = "predetermined-owner-identification-string";
   private static final String DESCRIPTION = "Club Description";
   private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
```

Comment:
Nit: Change "IDENTIFICATION_STRING" to "CLUB_ID"

Reply:
You're right, change has been made and pushed.